plugins/oauth/hooks.js

```diff
@@ -62,6 +64,21 @@ module.exports = Object.assign({}, StoreMixin, {
     }]);
   },
 
+  dcosInstallCommand(instructions) {
+    let steps = instructions.split('&&');
+    steps.splice(-1, 0, ' \n dcos auth login ');
```

Comment:
Made some changes so we don't split a string and pass in an Array. I'm not sure a Mount would be more beneficial without code duplication. Do you have a particular example in mind @orlandohohmeier?